packages/frontend/workspace/src/blob/storage/indexeddb.ts

```diff
@@ -0,0 +1,32 @@
+import { createStore, del, get, keys, set } from 'idb-keyval';
+
+import type { BlobStorage } from '../engine';
+
+export const createIndexeddbBlobStorage = (
+  workspaceId: string
+): BlobStorage => {
+  const db = createStore(`${workspaceId}_blob`, 'blob');
+  const mimeTypeDb = createStore(`${workspaceId}_blob_mime`, 'blob_mime');
```

Comment:
I think we do not need to create another store for mine type. The mine type can be saved within the same value object in `xxx_blob`.
However this involves some migration efforts though.

Reply:
Good suggestion 
Here is compatible with the original implementation.